card/db/migrate_core_cards/data/more_bootstrap.json

```diff
@@ -0,0 +1,51 @@
+{
+  "url": "http://localhost:3000/export.json",
+  "timestamp": "2015-02-25 14:55:17 +0100",
+  "card": {
+    "name": "export",
+    "type": "Pointer",
+    "content": "[[style: glyphicons]]\n[[simple skin]]\n[[simple bootstrap skin]]\n[[Default Layout]]\n[[*all+*script]]\n[[*page header]]\n[[style: bootstrap compatible]]\n[[Old Layout]]",
+    "value": [
+      {
+        "name": "style: glyphicons",
+        "type": "SCSS",
```

Comment:
this is normal CSS right?  Any reason not to do it as type CSS?  Not a big deal, probably, other than the hundredths of a second spent processing it when there are style changes...